exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/statistics/NDVMergedStatistic.java

```diff
@@ -123,7 +134,20 @@ public void setOutput(MapVector output) {
       NullableBigIntVector vv = (NullableBigIntVector) outMapCol;
       vv.allocateNewSafe();
       if (colHLLHolder != null) {
-        vv.getMutator().setSafe(0, 1, colHLLHolder.cardinality());
+        /* Hass and Stokes Duj1 estimator. See IBM Research Report RJ 10025 for more details.
```

Comment:
I think this is spelled 'Haas'.   Also, I believe this research report was subsequently published in a conference or journal.  Please see if you can find the proper citation.

Reply:
Done